Editor/Resources/EditorWindow/GameLiftPlugin.cs

```diff
@@ -0,0 +1,204 @@
+        {
+            Application.OpenURL(Urls.AwsGameTechForums);
+        }
+
+        [MenuItem("Amazon GameLift/Help/Report Issues", priority = 202)]
+        public static void OpenReportIssues()
+        {
+            Application.OpenURL(Urls.GitHubAwsLabs);
+        }
+
+        private void CreateGUI()
+        {
+            _root = rootVisualElement;
+            _mVisualTreeAsset = UnityEngine.Resources.Load<VisualTreeAsset>("EditorWindow/GameLiftPlugin");
+            if (_mVisualTreeAsset == null)
+            {
+                return;
+            }
+
+            VisualElement uxml = _mVisualTreeAsset.Instantiate();
+            _root.Add(uxml);
+            ApplyText();
+
+            CoreApi = CoreApi.SharedInstance;
+            var awsCredentials = AwsCredentialsFactory.Create();
+            CreationModel = awsCredentials.Creation;
+            UpdateModel = awsCredentials.Update;
+            RefreshProfiles();
+            
+            var contentContainer = _root.Q(className: "main__content").Children().First();
+            var landingPage = new LandingPage(contentContainer);
+            _userProfilesPage = new AwsUserProfilesPage(contentContainer,this);
+
+            _tabButtons = _root.Query<Button>(className: TabButtonClassName).ToList();
+            _tabContent = _root.Query(className: TabContentClassName).ToList();
+
+            _tabButtons.ForEach(button => button.RegisterCallback<ClickEvent>(_ => { OpenTab(button.name); }));
+
+            OpenTab("Landing");
+        }
+
+        private void ApplyText()
+        {
+            var l = new ElementLocalizer(_root);
+            l.SetElementText("Landing", Strings.TabLanding);
+            l.SetElementText("Credentials", Strings.TabCredentials);
+            l.SetElementText("Anywhere", Strings.TabAnywhere);
+            l.SetElementText("EC2", Strings.TabEC2);
+            l.SetElementText("Help", Strings.TabHelp);
+        }
+
+        private void OpenTab(string tabName)
+        {
+            _tabContent.ForEach(page =>
+            {
+                if (!page.name.StartsWith(tabName))
+                {
+                    page.RemoveFromClassList(TabContentSelectedClassName);
+                }
+                else
+                {
+                    page.AddToClassList(TabContentSelectedClassName);
+                }
+            });
+
+            _tabButtons.ForEach(button =>
+            {
+                if (button.name != tabName)
+                {
+                    button.RemoveFromClassList(TabButtonSelectedClassName);
+                }
+                else
+                {
+                    button.AddToClassList(TabButtonSelectedClassName);
+                }
+            });
+        }
+        
+        public void RefreshProfiles()
+        {
+            UpdateModel.Refresh();
+            CurrentState.AllProfiles = UpdateModel.AllProlfileNames;
+        }
+        
+        public void SetupWrapper()
```

Comment:
I don't see this being referenced anywhere. If this is part of later changes, please add it in as part of a later PR when it is being referenced. That makes it much easier to review with a context of how it's being used.

Reply:
This is referenced within AwsCredentialsProfilePage.cs